com.unity.render-pipelines.universal/Shaders/Utils/StencilDeferred.shader

```diff
@@ -15,8 +15,11 @@ Shader "Hidden/Universal Render Pipeline/StencilDeferred"
     #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Core.hlsl"
     #include "Packages/com.unity.render-pipelines.universal/Shaders/Utils/Deferred.hlsl"
 
-    // XR not supported in 2020.1 preview
-    #define XR_MODE 0 // defined(USING_STEREO_MATRICES) && (defined(_POINT) || defined(_SPOT) || defined(_DIRECTIONAL))
+    #if defined(USING_STEREO_MATRICES) && (defined(_POINT) || defined(_SPOT) || defined(_DIRECTIONAL))
```

Comment:
Is it possible to use USING_STEREO_MATRICES instead?

Reply:
Fixed,